### ISSUE TITLE
Creating a link or button within a widget

### ISSUE THREAD

**Issue by truthjuicearchitect** [schurpf/skype-mobile-switcher#1]
Hi
is there anyway for this functionality to work if it was added to a widget?

at the moment it shows the entire syntax, instead of just Call Us!
[phone "tel:+441111111111"]Call Us![/phone]

**Comment by schurpf** [comment]
Yeah shortcodes don't get processed in widgets by default. Maybe this post helps: http://englishmike.net/2008/07/07/wordpress-quick-tips-3adding-a-shortcode-to-a-sidebar-widget/
